src/modules/CreateChannel/components/InviteUsers/index.tsx

```diff
@@ -103,28 +105,27 @@ const InviteUsers: React.FC<InviteUsersProps> = ({
           });
           return;
         }
-        if (selectedUserList.length > 0) {
```

Comment:
Nothing changed in this if block, but just removed the condition cause there'll be no case this `selectedUserList.length === 0` with [this logic](https://github.com/sendbird/sendbird-uikit-react/pull/665/files#diff-08c2d4af32c32fa2cabfaf2e14278e24dea1f367e411a2483e84f38bdb73790cR97)